demos/k8s-demo/01_security_admin_steps

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+. ./utils.sh
+. ./admin_config.sh
```

Comment:
maybe worth a comment highlighting that this user has access to the admin config info?

Reply:
I changed the filename and added a comment to clarify this